src/Microsoft.Diagnostics.DebugServices.Implementation/RuntimeService.cs

```diff
@@ -32,7 +31,8 @@ public class RuntimeService : IRuntimeService, IDataReader, IExportReader
         public RuntimeService(ITarget target)
         {
             _target = target;
-            _onFlushEvent = target.OnFlushEvent.Register(() => {
+            _onFlushEvent = target.OnFlushEvent.Register(() =>
+            {
```

Comment:
I really hate this rule on lambdas.  I wish we wouldn't enforce it.

Reply:
Done